shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Value.hs

```diff
@@ -0,0 +1,258 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-unused-binds #-}
+{-# OPTIONS_GHC -fno-warn-missing-signatures #-}
```

Comment:
can we just add signatures and remove this?

Reply:
Yes I will clean this up. This PR was mostly for comments.